src/coreclr/src/jit/optcse.cpp

```diff
@@ -1537,7 +1537,7 @@ class CSE_Heuristic
 
     unsigned               aggressiveRefCnt;
     unsigned               moderateRefCnt;
-    unsigned               enregCount; // count of the number of enregisterable variables
+    unsigned               enregCount; // count of the number of enregistered variables
```

Comment:
Nit: these are not yet actually e registered, they are just predicted to be.

Reply:
OK